### PR TITLE
Pull Request - Index

### PR DESCRIPTION
**Pull request by Stateria**

### CODE REVIEW

**Review by eduardonacion — APPROVED** [review]
NICE COCK

**Comment by garett09** [comment]
Improve the code.

**Comment by garett09** [comment]
Reopen: check some issues and remove some unnecessary codes

File: css/sample.css [hunk 1]
```diff
@@ -1,57 +0,0 @@
-/**
```
Comment (garett09) [inline css/sample.css]: Do not delete: this is for sample

File: index.html [hunk 2]
```diff
@@ -1,42 +1,92 @@
...
-        <p>It uses utility classes for typography and spacing to space content out within the larger container.</p>
-        <p class="lead"> <a class="btn btn-primary btn-lg" href="#" role="button">Learn more</a> </p>
+<head>
+<meta charset="UTF-8" />
+<meta http-equiv="X-UA-Compatible" content="IE=edge" />
+<meta name="viewport" content="width=device-width, initial-scale=1.0" />
+<title>AFK Apartelle</title>
+<!-- Bootstrap LINKS -->
+
+     <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.0/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-wEmeIV1mKuiNpC+IOBjI7aAzPcEZeedi5yW5f2yOq55WWLwNGmvvx4Um1vskeMj0" crossorigin="anonymous">
```
Comment (garett09) [inline index.html:10]: fix formatting

File: index.html [hunk 3]
```diff
@@ -1,42 +1,92 @@
...
+  </button>
+  <button class="carousel-control-next" type="button" data-bs-target="#carouselExampleControls" data-bs-slide="next">
+    <span class="carousel-control-next-icon" aria-hidden="true"></span>
+    <span class="visually-hidden">Next</span>
+  </button>
+</div>
+	
+	
+	
+<!-- Bootstrap LINKS -->
```
Comment (garett09) [inline index.html:95]: fix formatting

File: index.html (hunk 2 above)
Comment (garett09) [inline index.html:10]:
```suggestion
     <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.0/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-wEmeIV1mKuiNpC+IOBjI7aAzPcEZeedi5yW5f2yOq55WWLwNGmvvx4Um1vskeMj0" crossorigin="anonymous">
```
Kindly review to this 
<link
      href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.0/dist/css/bootstrap.min.css"
      rel="stylesheet"
      integrity="sha384-wEmeIV1mKuiNpC+IOBjI7aAzPcEZeedi5yW5f2yOq55WWLwNGmvvx4Um1vskeMj0"
      crossorigin="anonymous"
    />

File: index.html [hunk 4]
```diff
@@ -1,42 +1,92 @@
...
+	<script src="https://cdn.jsdelivr.net/npm/@popperjs/core@2.9.2/dist/umd/popper.min.js" integrity="sha384-IQsoLXl5PILFhosVNubq5LC7Qb9DXgDA9i+tQ8Zj3iwWAwPtgFTxbJ8NT4GN1R8p" crossorigin="anonymous"></script>
```
Comment (garett09) [inline index.html:96]: Kindly copy the same formatting as line 10.

File: js/sample.js [hunk 5]
```diff
@@ -1,373 +0,0 @@
-/**
```
Comment (garett09) [inline js/sample.js]: Do note remove### #

**Review by garett09 — CHANGES_REQUESTED** [review]
Kindle check the commits #2 

- [ check index.html] 
@Stateria

**Review by garett09 — APPROVED** [review]
Medyo deliks lang yung line 10 ng index.php. Will take note kapag nag deploy kung gumana. elsem edit ko nalang

**Pull request closed without merging**